testing-daml-models/daml-scripts/2_first_script.md

```diff
@@ -0,0 +1,45 @@
+Next to a fixed `operator` party we want to allocate a list of parties given as an argument to the
+`initialize` script. We want them to follow the operator and the operator should follow them back.
+As before, we will be using the `allocatePartyWithHint` function. Replace the initialize script
+with:
+
+<pre class="file" data-target="clipboard">
+initialize: [Text] -> Script ()
+initialize initialParties = do
+    operator <- allocatePartyWithHint "operator" (PartyIdHint "operator")
+    ps <- forA initialParties $ \partyStr -> do
+      allocatePartyWithHint partyStr (PartyIdHint partyStr)
+    debug "done!"
+</pre>
+
+Easy! Scripts are just like usual DAML functions and follow the same syntax to declare arguments and
+type signatures.
+
+1. Now we get the other party names from the `initialParties` argument to the script. 
+1. `forA: (Applicative m) => [a] -> (a -> m b) -> m [b]` starts a `for loop` that you can
```

Comment:
Maybe add one more line between the two points as the way it renders it's hard to tell what's 1) and what's 2)

![image](https://user-images.githubusercontent.com/56260209/81648301-ca337d80-942e-11ea-94f9-b9335374b60a.png)

Reply:
added a blank line.